app/assets/stylesheets/components/_personal-key.scss

```diff
@@ -2,7 +2,7 @@
   position: relative;
 
   &::before {
-    background-image: url('p-key.svg');
+    background-image: url('personal-key/personal-key.svg');
     background-repeat: no-repeat;
     content: '';
     height: 60px;
```

Comment:
Yes, good catch. However I'm not entirely satisfied with this method of sizing things. I think we should have a variable somewhere for the "default" icon size, then down the road add other relative sizes (small, medium, etc) as variables should the need arise. What do you think?

Reply:
Hm, that could work. It would be nice if we could move toward something more self-contained, vs. resorting to a global variable because of how these styles are scattered throughout various stylesheets.

Thinking something like a component (which might require markup revisions) or Sass mixin.

Reply:
By component do you mean custom element, react component, or rails view component (or any of them)? At one point I _was_ thinking of adding a rails template helper function to generate the markup for icons in some standard way, but decided that it would add too much indirection and more code

Reply:
> By component do you mean custom element, react component, or rails view component (or any of them)? At one point I _was_ thinking of adding a rails template helper function to generate the markup for icons in some standard way, but decided that it would add too much indirection and more code

I was being intentionally vague, but I suppose that's not very helpful of me 😅 I think it could be any of the above, or even just a CSS class that we apply, where the definition of those CSS files are contained in a single stylesheet.

For example:

```html
<div class="alert-icon alert-icon--personal-key alert-icon--small"></div>
```

(Might warrant more thought on if/how to make these accessibly labelled)